docs/pages/upgrading/cloud-kubernetes.mdx

```diff
@@ -4,127 +4,111 @@ description: Provides instructions for upgrading Teleport Cloud agents that run
 ---
 
 This guide explains how to upgrade Teleport Cloud agents running on Kubernetes.
-On Teleport Cloud, Auth Service and Proxy Service upgrades are managed for you.
-To keep agents up to date, you can either enroll them in automatic updates or
-upgrade them manually using the method you used to install Teleport.
+To keep agents up to date, you can either perform:
+- [Automatic updates](#automatic-updates) (recommended)
+- [Manual updates](#manual-updates)
```

Comment:
If we recommend automatic updates, I would make it easier for users to enable automatic updates and more difficult to enable manual updates. We should frame this guide around automatic updates and describe manual updates as an optional users can take if automatic updates are not possible.

Reply:
We want to push users to use automatic updates, but there are still many users that are not ready or do not want to use on automatic updates. For these users, if we make it difficult to perform manual updates, I think they will be more likely to just not upgrade their agents/clusters.

I'd like to make the instructions for both methods easy to follow, but make it clear that automatic updates is the better method. And to highlight that if they opt out of automatic updates, they need to perform updates weekly, or at the very least for every major version release.